examples/advanced/llm_hf/requirements.txt

```diff
@@ -0,0 +1,8 @@
+nvflare
+torch
+datasets
+tensorboard
+transformers
+peft
+trl
+flash-attn
```

Comment:
Do we need to add the license for flash-attn to the [3rd party folder](https://github.com/NVIDIA/NVFlare/tree/main/3rdParty)?

Reply:
thanks! added